### PR TITLE
Travis-ci and Coveralls added

### PR DESCRIPTION
**Pull request by helloimcarmine**
In this pull request I have added Travis Ci and Coveralls to this repository, showing two new badges in the README.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.4"
+  - "3.5"
+install: pip install coveralls
+ # - if [ "$TRAVIS_PYTHON_VERSION" == "3.2" ]; then travis_retry pip install coverage==3.7.1; fi
+ # - if [ "$TRAVIS_PYTHON_VERSION" != "3.2" ]; then travis_retry pip install coverage; fi
```
Comment (bassosimone) [inline .travis.yml]: could you remove the two commented out lines above, please?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Neubot runtime
 
+[![Build Status](https://travis-ci.org/neubot/neubot-runtime.svg?branch=master)](https://travis-ci.org/neubot/neubot-runtime)
+
+[![Coverage Status](https://coveralls.io/repos/neubot/neubot-runtime/badge.svg?branch=master)](https://coveralls.io/r/neubot/neubot-runtime?branch=master)
+
+
```
Comment (bassosimone) [inline README.md:4]: could you remove this extra empty line, please?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+script: coverage run neubot_runtime/test_utils_path.py
```
Comment (bassosimone) [inline .travis.yml]: could you find a way to teach coverage to run all the test scripts present below the neubot_runtime directory?

**Comment by bassosimone** [comment]
Hi @helloimcarmine and thanks! I have already looked at this and find it good. You should address my comments above, and then I will happily merge.

File: .travis.yml (hunk 1 above)
Comment (helloimcarmine) [inline .travis.yml]: Done!

File: README.md (hunk 2 above)
Comment (helloimcarmine) [inline README.md:4]: Done!

File: .travis.yml (hunk 3 above)
Comment (helloimcarmine) [inline .travis.yml]: Done!

**Comment by bassosimone** [comment]
Ah, right, I forgot about the failing test. We'll, I'd say we fix together the failing test on Monday and then we proceed with merging.

**Comment by bassosimone** [comment]
Thanks!

**Pull request merged**